samples/KvsVideoOnlyStreamingSample.c

```diff
@@ -91,8 +93,10 @@ INT32 main(INT32 argc, CHAR *argv[])
     CHK_STATUS(createDefaultDeviceInfo(&pDeviceInfo));
     // adjust members of pDeviceInfo here if needed
     pDeviceInfo->clientInfo.loggerLogLevel = LOG_LEVEL_DEBUG;
+    pDeviceInfo->storageInfo.storageSize = DEFAULT_STORAGE_SIZE;
```

Comment:
Is setting this necessary. Is it not possible to do the calculation with the 128MB default?

Reply:
I think people reading the sample wouldnt know about the 128MB default. Also this way they can change the storage size and the buffer duration would still be right.